.github/workflows/test.yml

```diff
@@ -506,6 +508,8 @@ jobs:
           make -C test control-plane WHAT="External Gateway"
         elif [ "${{ matrix.target }}" == "kv-live-migration" ]; then
           make -C test control-plane WHAT="Kubevirt Virtual Machines"
+        elif [ "${{ matrix.target }}" == "control-plane-helm" ]; then
+          make -C test control-plane WHAT='Pod to '
```

Comment:
Open for discussion topic: Currently, this PR executes all unit tests and a minimal set of e2e tests, as demonstrated by what here, that checks pod-to-pod communication. This consistently passes. However, there are still issues with executing all other control-plane tests, which must be addressed in the near future. What needs to be decided is whether we should block this PR from merging until then or address the remaining issues in subsequent PRs. 
/cc @girishmg, @trozet, @tssurya, and @jcaamano

Reply:
IMHO i would rather merge it with those tests failing, than being skipped. That is more likely to motivate someone to go fix it. @flavio-fernandes did you figure out the root cause of the egress firewall failure?

Reply:
Hey, interesting take, @trozet. Discounting the flakes, there are still quite a number of tests failing—almost too many, which could demotivate people from fixing them. :) Joke aside, merging something we know is failing seems worse than not merging, in my humble opinion. How about a compromise: would you be okay if we explicitly skipped the tests known to be broken in a well-defined 'helm' block in [e2e-cp.sh](https://github.com/ovn-org/ovn-kubernetes/blob/master/test/scripts/e2e-cp.sh)? On the same block, we could list the issue to track them.

To answer your question, I forgot to look at the egress failures after we talked. My bad. I was overwhelmed by the other tests that were consistently failing and never went back to it.

All in all, let me at least categorically list each non-flake test failure and create issues to track them before merging this PR.

Reply:
@flavio-fernandes I would at least get to the bottom of why tests are failing? Understand the failures and have a plan for it.

what if the tests are failing because the helm chart forgot to set some environment variables in the daemonset or deployment?

Reply:
great point @girishmg : @flavio-fernandes did you ensure all the feature's custom crds are correctly installed? example: EIP, EFW, ANP,BANP all these crds are installed on the cluster? that seems like the helm lane bug to begin with which is probably why EFW also failed, I am saying this based on this error: `no matches for kind "AdminNetworkPolicy" in version "policy.networking.k8s.io/v1alpha1"  ensure CRDs are installed first` that is something we should ensure the helm install is doing.. (doing an oc get crd on a helm installed cluster should tell you, the kind install path does this, where it oc applies all the stuff in the dist/template file)

Reply:
@tssurya we are definitely missing 2 admin netw policy crds. They were supposed to be in the helm/ovn-kubernetes/crds directory. Other failures seem to be unrelated to crds, but I will double check.
```
helm/ovn-kubernetes/crds/policy.networking.k8s.io_adminnetworkpolicies.yaml
helm/ovn-kubernetes/crds/policy.networking.k8s.io_baselineadminnetworkpolicies.yaml
```

Reply:
**Update:** I compiled a list with all the non flakes, as well as simple commands to reproduce it locally in this doc: 
https://docs.google.com/document/d/13BEpBq24zb7TFzx-jG18_cP-IROzybJE5Mbp5NyjyEU/edit?usp=sharing
Many cases are failing due to the status attribute on the cr not getting updated when it should. Will dig deeper into them next.

Reply:
**Update:** Issue with status attribute not being updated as expected is related to the fact that many ovnkube-master controllers were not being enabled. Fixing it! 
ref:  https://github.com/ovn-org/ovn-kubernetes/pull/4238/commits/d2484a5b98ae158e557827eafa051583aac12237

Reply:
that's awesome work @flavio-fernandes ! So I am ok as long as for all failures we open github issue trackers listing them as TODO, example I see eqos ones failing now so we open issue and skip then and then can do new prs to fix them (that's the approach we took for v6 flakes as well); so that way if every skip has a link to github issue we can track them better